ui/branding/semantic-ui/less/globals/site.overrides

```diff
@@ -1,34 +1,34 @@
-//
-// overrides for css styles. To reference a font/image from the assets directory, use:
-// url("~@less/fonts/your-font-name") or url("~@less/images/your-image-name")
-//
+// //
+// // overrides for css styles. To reference a font/image from the assets directory, use:
+// // url("~@less/fonts/your-font-name") or url("~@less/images/your-image-name")
+// //
 
 @font-face {
   font-family: "Roboto";
   font-style: normal;
   font-weight: 400;
-  src: url("../../../fonts/Roboto-Regular.ttf") format("truetype");
+  src: url("../fonts/Roboto-Regular.ttf") format("truetype");
```

Comment:
Use the new `~@less` alias to reference font & image assets here (should be more reliable than relative paths)